### PR TITLE
Unit test setup and cleanup

### PR DESCRIPTION
**Pull request by zubhav**

### CODE REVIEW

**Comment by zubhav** [comment]
#### 📝Changed routes:


| [**/_layout**](https://covid-19-git-unit-test-setup.prubby.now.sh/_layout) | [**/**](https://covid-19-git-unit-test-setup.prubby.now.sh/) |
|:-:|:-:|
|<a href="https://covid-19-git-unit-test-setup.prubby.now.sh/_layout">Screenshot of /_layout</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/BR4LMZhkXNwUeIKtRpNAt35BR6o_.png">(view full size)</a> |<a href="https://covid-19-git-unit-test-setup.prubby.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/26Pjrsj74T2H4HOZir_6MBBG1mQ_.png">(view full size)</a> |







Commit d85ed721db68a1e8b0acc4a1d3e4f206d859173d (https://covid-19-git-unit-test-setup.prubby.now.sh).

File: src/components/Card/__tests__/Card.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        value: 'open',
+                    },
+                ],
+                selected: 'close',
+                current: 1,
+            })
+
+            expect(getByText('$ABCD')).toBeInTheDocument()
+            expect(getByText('Close (selected)')).toBeInTheDocument()
+            expect(getByText('2')).toBeInTheDocument()
```
Comment (pranjaljately) [inline src/components/Card/__tests__/Card.test.js:27]: What is this checking?

Comment (pranjaljately) [reply src/components/Card/__tests__/Card.test.js:27]: Okay correct me if I am wrong - this is checking the close value based on the 'current' index...

Comment (zubhav) [reply src/components/Card/__tests__/Card.test.js:27]: yep

**Pull request merged**